### PR TITLE
simplenote: 1.1.3 -> 1.8.0

### PR DESCRIPTION
**Pull request by Kiwi**
###### Motivation for this change
I wanted to take a look at Simplenote and the first thing it did was say there was a new version I needed to download manually (1.3.0, iirc). So I check the website, and it's up to 1.8.0! And then I checked nixpkgs issues to make sure nobody else had already submitted one before I wasted time (wouldn't be the first time, not even the first time this week...) and the latest [was an unmerged 1.5.0 from April](https://github.com/NixOS/nixpkgs/pull/59418)

###### Things done
I did not run nix-review because it is [broken for me right now??](https://gist.github.com/Kiwi/b36e78849c3064eda892180c8690194b)

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @worldofpeace

### CODE REVIEW

File: pkgs/applications/misc/simplenote/default.nix [hunk 1]
```diff
@@ -1,64 +1,77 @@
...
+    runtimeDependencies = [ udev.lib ];
+
+    postFixup = ''
+      ls -ahl $out
+      makeWrapper $out/opt/Simplenote/simplenote $out/bin/simplenote \
+      "''${gappsWrapperArgs[@]}"
+    '';
+  };
+
+in if stdenv.isDarwin then
```
Comment (jonringer) [inline pkgs/applications/misc/simplenote/default.nix]: the meta.platforms section should be evaluated which will yield that darwin isn't supported

**Comment by matthewbauer** [comment]
@grahamcofborg eval

File: pkgs/applications/misc/simplenote/default.nix (hunk 1 above)
Comment (Mic92) [reply pkgs/applications/misc/simplenote/default.nix]: As @jonringer noted. `meta.platforms` is already set to `"x86_64-linux"`, which is sufficient.

**Review by worldofpeace — APPROVED** [review]

**Comment by worldofpeace** [comment]
Thanks for updating this @Kiwi :heart:

**Pull request merged**